client/src/ui/App.tsx

```diff
@@ -2,6 +2,29 @@ import React, { Component } from 'react';
 import SearchBar from './SearchBar.jsx';
 import "./css/App.css";
 
+var date = new Date();
+var month = date.getMonth();
+var hours = date.getHours();
+var monthclass = "";
+var dayclass = "afternoon";
+
+if (month < 2 || month > 10) {
+  monthclass = "winter"
+}
+
+else if (month > 6) {
+  monthclass = "fall"
+}
+
+if (hours < 5 || hours > 19) {
+  dayclass = "night"
+  monthclass = "";
```

Comment:
Is it right to do `monthclass = "";` here when the if statement checks for hours? Perhaps you forgot to delete this?

Reply:
The reason I set monthclass to the empty string when it is night is that in the figma, there is no design for winter night, only designs for night and winter separately. And combining the night gradient with the winter illustration did not look right because of the bright snow vs the dark sky. So essentially when the time of day is night, it overrides the season illustration and just uses the night illustration, so I set the monthclass to the empty string to only use the night illustration and gradient to avoid the conflicting design.

Reply:
Sounds good